doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,296 @@
+.. _strict-mode:
+
+===========
+Strict Mode
+===========
+
+This mode is designed to avoid problems with improper configuration of |PXC|.
+It performs a number of validations against essential features of |PXC|.
+Some validations are performed only at startup,
+while others remain enabled during runtime.
+
+Depending on the actual mode you select,
```

Comment:
Wondering why it is inserting newline for for such a short text.
May be it looks ok in html format but looks bit weird here.